charts/opentelemetry-demo/values.yaml

```diff
@@ -737,7 +737,7 @@ grafana:
         - name: Jaeger
           uid: webstore-traces
           type: jaeger
-          url: 'http://{{ include "otel-demo.name" . }}-jaeger:16686'
+          url: 'http://{{ include "otel-demo.name" . }}-jaeger:16686/jaeger/ui'
```

Comment:
This should be 
```
          url: 'http://{{ include "otel-demo.name" . }}-jaeger-query:16686'
```

When we moved to use the Jaeger sub-chart, the query service name changed. We had to do a similar change for other spots, and missed this one.

Reply:
@puckpuck You are right, the svc name should be "jaeger-query". But, if the url missing "/jaeger/ui" path, it will not work out of the box.